pyproject.toml

```diff
@@ -7,3 +7,59 @@ extend-exclude = '''
   | devenv
 )/
 '''
+
+[tool.vendoring]
+destination = "conda/_vendor/"
+requirements = "conda/_vendor/vendor.txt"
+namespace = "conda._vendor"
+
+protected-files = ["__init__.py", "vendor.txt", "README.rst"]
+patches-dir = "tools/vendoring/patches"
+
+[tool.vendoring.transformations]
+substitute = []
+drop = [
+  # contains unnecessary scripts
+  "bin/",
+  # remove unwanted modules
+  '^boltons/(cache|debug|depr|dict|easter|eco|exc|file|format|func|gc|io|iter|json|list|math|mbox|named|path|queue|socket|stats|str|table|tb||type|url)utils\.py',
+  "boltons/txurl_notes.md",
+  "cpuinfo/__main__.py",
+  "tlz/",
+  '^toolz/(_signatures|functoolz)\.py',
+  "toolz/curried",
+  "toolz/sandbox",
+  "toolz/tests",
```

Comment:
Auxlib, toolz and boltons seem to be way too many extra batteries vendored than make sense IMO.